backend/services/deployments/app/app_releases.go

```
@@ -141,3 +141,21 @@ func (d *Deployments) DeleteReleases(
 	err = d.db.DeleteReleasesByNames(ctx, releaseNames)
 	return ids, err
 }
+
+func (d *Deployments) GetRelease(
+	ctx context.Context,
+	releaseName string,
+) (*model.Release, error) {
+	release, err := d.db.GetRelease(ctx, releaseName)
+	switch err {
+	case nil:
+		return release, nil
+	case store.ErrNotFound:
+		return nil, ErrReleaseNotFound
+	default:
+		// Rewrite internal errors
+		log.FromContext(ctx).
+			Errorf("failed to get release from database: %s", err.Error())
+		return nil, ErrModelInternal
```

Why are we masking this error? It is the first time I see it.

---

It's the pattern we use in the app_releases, so I just applied it here without much thinking, tbh.
I wasn't the one to introduce it originally, maybe @merlin-northern knows something more?

---

it is a good idea to do it. you never know what internal information the error will carry. I would keep it.